Makefile.am

```diff
@@ -2118,10 +2118,20 @@ git-archive:
 # How will we update this to python 3?
 
 lib389: src/lib389/setup.py
-	cd $(srcdir)/src/lib389; $(PYTHON) setup.py build ; $(PYTHON) setup.py build_manpages
```

Comment:
Mock build fails for me here (in 1minutetip).

```
make -C ./src/lib389
make[2]: Entering directory '/builddir/build/BUILD/389-ds-base-2.1.0.20211103git691579332/src/lib389'
/usr/bin/python setup.py build
make[2]: /usr/bin/python: No such file or directory
```

I think the Mock build env has only '/usr/bin/python3' (which is used in the core Makefile.am). I haven't checked deeper though if we need to change the way we run Mock.